cookbooks/bcpc-hadoop/templates/default/hdp_hdfs-site.xml.erb

```diff
@@ -15,6 +15,11 @@
   </property>
 
   <property>
+    <name>dfs.datanode.balance.bandwidthPerSec</name>
```

Comment:
Could this have a node attribute too? (100MB is a drop in the bucket for nodes with 20 and 40GB throughput.)